test/lib/code_corps_web/controllers/github_event_controller_test.exs

```diff
@@ -28,13 +28,18 @@ defmodule CodeCorpsWeb.GithubEventControllerTest do
     end
 
     @tag authenticated: :admin
-    test "lists all entries on index", %{conn: conn} do
-      [github_event_1, github_event_2] = insert_pair(:github_event)
+    test "lists all entries on index by inserted_at", %{conn: conn} do
+      github_event_1 = insert(:github_event, inserted_at: Timex.now())
+      github_event_2 = insert(:github_event, inserted_at: Timex.now() |> Timex.shift(days: 3))
 
-      conn
+      data = conn
```

Comment:
Since it looks like we do need to do this extra work here, I think the convention is:

```
data =
  conn
  |> my_func()
```